src/main/webapp/components/BuildSnapshot.js

```diff
@@ -1,14 +1,72 @@
-import * as React from "react";
+import * as React from 'react';
+import {getFields} from './utils/getFields.js';
 
 /**
  * Component used to render Build Information.
  *
  * @param {*} props input properties containing build information
```

Comment:
please list the expected keys in these props. They are implicit in the constants inside the body but it would be useful to not have to read it. Specifically, props itself is expected to have a `buildData` dict with further header and tray fields, right?

Reply:
Updated the comment to describe the expected types of the params, as well as some of the expected keys. 9f6231c